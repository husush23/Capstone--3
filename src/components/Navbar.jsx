import React from 'react';
import '../styles/navbar.css';

function Navbar() {
  return (
    <nav>
      <div />
      <div className="p-2">
        <span>
          <i className="fa-solid fa-microphone-lines" />
        </span>
        <span>
          <i className="fa-solid fa-gear" />
        </span>
      </div>
    </nav>
  );
}

export default Navbar;
