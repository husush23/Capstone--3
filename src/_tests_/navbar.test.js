/*eslint-disable*/
import Navbar from '../components/Navbar';
import React from 'react';
import {render} from '@testing-library/react';

describe('Navbar', () => {
  test('Renders Navbar correctly', () => {
    render(<Navbar />);
  });
  test('renders Navbar correctly', () => {
    const {asFragment} = render(<Navbar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
