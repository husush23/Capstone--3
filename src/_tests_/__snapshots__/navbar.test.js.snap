// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders Navbar correctly 1`] = `
<DocumentFragment>
  <nav>
    <div />
    <div
      class="p-2"
    >
      <span>
        <i
          class="fa-solid fa-microphone-lines"
        />
      </span>
      <span>
        <i
          class="fa-solid fa-gear"
        />
      </span>
    </div>
  </nav>
</DocumentFragment>
`;
